RubberduckTests/Inspections/LineLabelNotUsedInspectionTests.cs

```diff
@@ -0,0 +1,260 @@
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Rubberduck.Inspections.Concrete;
+using Rubberduck.Inspections.QuickFixes;
+using Rubberduck.Parsing.Inspections.Resources;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using RubberduckTests.Mocks;
+
+namespace RubberduckTests.Inspections
+{
+    [TestClass]
+    public class LineLabelNotUsedInspectionTests
+    {
+        [TestMethod]
+        [TestCategory("Inspections")]
+        public void LabelNotUsed_ReturnsResult()
+        {
+            const string inputCode =
+@"Sub Foo()
+label1:
+End Sub";
+
+            IVBComponent component;
+            var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out component);
+            var state = MockParser.CreateAndParse(vbe.Object);
+
+            var inspection = new LineLabelNotUsedInspection(state);
+            var inspectionResults = inspection.GetInspectionResults();
+
+            Assert.AreEqual(1, inspectionResults.Count());
+        }
+
+        [TestMethod]
+        [TestCategory("Inspections")]
+        public void LabelNotUsed_ReturnsResult_MultipleLabels()
+        {
+            const string inputCode =
+@"Sub Foo()
+label1:
+label2:
+End Sub";
+
+            IVBComponent component;
+            var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out component);
+            var state = MockParser.CreateAndParse(vbe.Object);
+
+            var inspection = new LineLabelNotUsedInspection(state);
+            var inspectionResults = inspection.GetInspectionResults();
+
+            Assert.AreEqual(2, inspectionResults.Count());
+        }
+
+        [TestMethod]
+        [TestCategory("Inspections")]
+        public void LabelUsed_DoesNotReturnResult()
```

Comment:
A better name would indicate that the input *does* include a `GoTo` jump that refers to the label, e.g. `GivenGoToStatement_LabelNotUsed_YieldsNoResult` or similar.